proofs/tree.go

```diff
@@ -130,6 +139,9 @@ func (doctree *DocumentTree) CreateProof(prop string) (proof proofspb.Proof, err
 		return proofspb.Proof{}, err
 	}
 	salt, err := getByteValueByProperty(prop, doctree.salts)
+	if err != nil {
+		return proofspb.Proof{}, err
```

Comment:
Can we use named returns so that we don't have to create `proofspb.Proof{}` for every return?